tests/MongoDB.Driver.Tests/Linq/Translators/PredicateTranslatorTests.cs

```diff
@@ -635,7 +691,7 @@ public void ClassEquals()
             Assert(
                 x => x.C == new C { D = "Dexter" },
                 0,
-                "{ C : { Ids : null, D : 'Dexter', E : null, S : null, X : null } }");
+                "{ C : { Ids : null, D : 'Dexter', E : null, S : null, X : null, Y : null, Z : null } }");
```

Comment:
No real changes. It needs to be updated just because the test class structure has been updated and this test is depended on it

Reply:
All the below changes are the same as the current.